wear/src/main/java/io/homeassistant/companion/android/home/views/SensorUi.kt

```diff
@@ -21,13 +24,43 @@ fun SensorUi(
     basicSensor: SensorManager.BasicSensor,
     onSensorClicked: (String, Boolean) -> Unit,
 ) {
+    var checked = sensor?.enabled == true
+
+    val backgroundRequest =
+        rememberLauncherForActivityResult(ActivityResultContracts.RequestPermission()) {}
+
+    val permissionLaunch = rememberLauncherForActivityResult(
+        ActivityResultContracts.RequestMultiplePermissions()
+    ) { isGranted ->
+        var allGranted = true
+        isGranted.forEach {
+            if (manager.requiredPermissions(basicSensor.id)
+                .contains(Manifest.permission.ACCESS_FINE_LOCATION) && manager.requiredPermissions(basicSensor.id)
+                    .contains(Manifest.permission.ACCESS_BACKGROUND_LOCATION) && it.key == Manifest.permission.ACCESS_FINE_LOCATION
```

Comment:
Splitting conditions with `contains` over two lines is very confusing, any reason why you did this?
```
                                                   manager.requiredPermissions(basicSensor.id)
.contains(Manifest.permission.ACCESS_BACKGROUND_LOCATION)
```

Reply:
> Splitting conditions with `contains` over two lines is very confusing, any reason why you did this?

yea android studio split it up like that and for some reason I kept it lol